Metric/Metric.php

```diff
@@ -140,34 +139,28 @@ public function getResolvedTags(array $resolvers = []): array
 
         // Add global parameters (configured in client or group)
         if (is_array($this->configurationTags)) {
-            if ($this->event instanceof KernelEvent && $this->event->isMasterRequest()) {
```

Comment:
so that's what was supposed to fix what you're fixing now, kinda surprised symfony broke something like that

Reply:
The problem of this fix is that you don't always have a kernel event sent.
So if you want to resolve the tags for the other metrics, you lost the request context :/

Reply:
only some kernel events had the sack purged

Reply:
This "tag resolution" must be on a higher level in order to resolve the configuration tags for every sent metric